packages/nextjs-kit/src/lib/sortChar.ts

```diff
@@ -0,0 +1,29 @@
+import { DataToSort } from '../types';
+
+/**
+ *
+ * @param {DataToSort['data']} sortObj.data - The data sort
+ * @param {DataToSort['key']} sortObj.key - Key to sort object by
+ * @param  {DataToSort['direction']} sortObj.direction - Direction to sort the data
+ * @returns {Record<string, string | number>[]} An array of data sorted by the given key and direction
```

Comment:
OK I was very wrong here, apologies!
We don't need the type at all when we are writing in TypeScript with TSDoc.
I was confused because we do need it with JSDoc.
My bad! I can do the change for these.